store/messagebus_queuesettings.yaml

```diff
@@ -0,0 +1,37 @@
+import:
```

Comment:
Can we split queuesettings (referring to name of the file) so that it is properly converted to CamelCase?